src/firmware/shared/BUILD

```diff
@@ -24,6 +24,11 @@ cc_library(
     srcs = ["util.c"],
     hdrs = ["util.h"],
     deps = [
-        "//firmware/main/math:tbots_math",
+        ":tbots_math",
```

Comment:
tbots_math is weird name->how is tbots math different from normal math? Can we think of a new name?

Reply:
We can't use `math` because of naming conflicts (please see declaration of `tbots_math`), totally happy if we want to rename it, but definitely outside the scope of this PR.